### PR TITLE
Fix broken link to readthedocs

### PR DESCRIPTION
**Pull request by jeffkala**
Change version of readthedocs in the link on L9 to latest.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Nautobot is a Network Source of Truth and Network Automation Platform.
 Nautobot runs as a web application atop the [Django](https://www.djangoproject.com/)
 Python framework with a [PostgreSQL](https://www.postgresql.org/) database. 
 
-The complete documentation for Nautobot can be found at [Read the Docs](https://nautobot.readthedocs.io/en/stable/).
+The complete documentation for Nautobot can be found at [Read the Docs](https://nautobot.readthedocs.io/).
```
Comment (jathanism) [inline README.md:10]: This should link to https://nautobot.readthedocs.io/en/latest/

File: README.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ or join us in the **#nautobot** Slack channel on [Network to Code](https://netwo
 
 ## Installation
 
-Please see [the documentation](https://nautobot.readthedocs.io/en/latest/) for
+Please see [the documentation](https://nautobot.readthedocs.io/) for
```
Comment (jathanism) [inline README.md:36]: This should link to https://nautobot.readthedocs.io/en/latest/installation/

**Review by jathanism — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jeffkala) [reply README.md]: fixed.

File: README.md (hunk 2 above)
Comment (jeffkala) [reply README.md]: fixed.

**Review by jathanism — APPROVED** [review]

**Pull request merged**